client/src/components/utils/CalculateScore.js

```diff
@@ -70,12 +64,19 @@ const CalculateScore = (quiz_id, respondent_id) => {
 				gradedQuestions.push(question_ans);
 				console.log("graded");
 				console.log(gradedQuestions);
+			} else {
+				question_ans = {};
+				question_ans["answerGiven"] = [];
+				question_ans["question_id"] = question_id;
```

Comment:
Is this section for when answers are blank/null or when they are wrong?

Reply:
this is when user submits no answer (in case time runs out) for certain questions, there is no array element saved, thus it is manually creating quiz id, answergiven as null array, and 0 points since nothing is answered.